syncd/syncd.cpp

```diff
@@ -1715,6 +1715,7 @@ void handlePortMap(const std::string& portMapFile)
         std::string name, lanes, alias;
         iss >> name >> lanes >> alias;
 
+        iss.clear();
```

Comment:
many examples I found recommend 
- use
- clear
- reuse
and I tested this code it works fine
is there any special reason to change this? it will take several hours to rebuild and test a new code...